Cargo.toml

```diff
@@ -117,6 +117,7 @@ sqlx = { version = "0.8.5", default-features = false, features = [
     "uuid",
     "chrono",
 ] }
+test-log = "0.2.18"
```

Comment:
We already have `tracing_test`, it seems to me that this does something similar.
Do we need both?